sqlglot/dialects/snowflake.py

```diff
@@ -107,6 +107,13 @@ def _builder(args: t.List) -> E:
     return _builder
 
 
+def _build_bitor(args: t.List) -> exp.BitwiseOr | exp.Anonymous:
+    if len(args) == 3:
+        return exp.Anonymous(this="BITOR", expressions=args)
+
+    return exp.BitwiseOr(this=seq_get(args, 0), expression=seq_get(args, 1))
+
```

Comment:
It might look a bit cleaner if we did `binary_from_function(exp.BitwiseOr)(args)` here

Reply:
Done @VaggelisD